src/main/java/frc/trigon/robot/motorsimulation/SingleJointedArmSimulation.java

```diff
@@ -0,0 +1,40 @@
+package frc.trigon.robot.motorsimulation;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+public class SingleJointedArmSimulation extends MotorSimulation {
+    private final SingleJointedArmSim armSimulation;
+
+    public SingleJointedArmSimulation(DCMotor motor, double gearRatio, double retractedArmLengthMeters, double minimumAngleRadians, double maximumAngleRadians, boolean simulateGravity, double armMass) {
+        armSimulation = new SingleJointedArmSim(motor, gearRatio, SingleJointedArmSim.estimateMOI(retractedArmLengthMeters, armMass), retractedArmLengthMeters, minimumAngleRadians, maximumAngleRadians, simulateGravity, minimumAngleRadians);
+    }
+
+    @Override
+    double calculateFeedforward(double ks, double kg, double kv, double ka) {
+        ArmFeedforward feedforward = new ArmFeedforward(ks, kg, kv, ka);
+        return feedforward.calculate(Units.rotationsToRadians(getPositionRevolutions()), Units.rotationsToRadians(getVelocityRevolutionsPerSecond()));
+    }
```

Comment:
1. I don't want you to create the feedforward class every time. Instead, do the calculation yourself.
2. Since feedforward is open-loop is accepts only the target position / vel. 
3. Make sure you're comparing in the unit of the conversion factor.